web/package-lock.json

```diff
@@ -1035,7 +1035,7 @@
         },
         "whatwg-fetch": {
           "version": "2.0.4",
-          "resolved": "https://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.4.tgz",
+          "resolved": "http://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.4.tgz",
```

Comment:
Does https://github.com/npm/npm/issues/20106#issuecomment-383770315 help?

Looks like we may need to upgrade to a newer version of nvm to permanently fix the problem. Please lmk if it works?

Reply:
It worked, thanks!